subprojects/language-groovy/src/integTest/groovy/org/gradle/groovy/GroovyDocIntegrationTest.groovy

```diff
@@ -112,39 +112,4 @@ class GroovyDocIntegrationTest extends MultiVersionIntegrationSpec {
         where:
         module << ['groovy']
     }
-
-    @Issue(["GRADLE-3174", "GRADLE-3463"])
-    def "Error in Groovydoc generation is logged"() {
```

Comment:
This is a multi-version integration spec, so this may only fail on older versions of Groovy.  I think we should keep this test and find a way to trigger the same kind of error on newer versions of Groovy.